skywalking-collector/skywalking-collector-worker/src/main/java/com/a/eye/skywalking/collector/worker/application/metric/TraceSegmentRecordMember.java

```diff
@@ -0,0 +1,139 @@
+package com.a.eye.skywalking.collector.worker.application.metric;
+
+
+import akka.actor.ActorRef;
+import com.a.eye.skywalking.collector.actor.AbstractASyncMemberProvider;
+import com.a.eye.skywalking.collector.actor.AbstractMember;
+import com.a.eye.skywalking.collector.actor.selector.RollingSelector;
+import com.a.eye.skywalking.collector.queue.MessageHolder;
+import com.a.eye.skywalking.collector.worker.PersistenceMember;
+import com.a.eye.skywalking.collector.worker.application.persistence.TraceSegmentRecordPersistence;
+import com.a.eye.skywalking.trace.Span;
+import com.a.eye.skywalking.trace.TraceSegment;
+import com.a.eye.skywalking.trace.TraceSegmentRef;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.lmax.disruptor.EventFactory;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author pengys5
+ */
+public class TraceSegmentRecordMember extends PersistenceMember {
```

Comment:
`TraceSegmentRecordMember` is a subclass of `PersistenceMember<T>`, which is a generics, why did not the subclass set the **T**?